apps/gitness/src/components-v2/app-shell.tsx

```diff
@@ -68,7 +71,7 @@ const AppShell = () => {
    * Map mock data menu by type to Settings and More
    */
   const { moreMenu, settingsMenu } = useMemo(() => {
-    const navbarMenuData = getNavbarMenuData(t)
+    const navbarMenuData = getNavbarMenuData(t, space_ref ?? spaces[0].path)
```

Comment:
`spaces[]` might be empty if the API call is not complete yet, so `spaces[0]` can throw an exception.
we should add a length check on `spaces`